pynio/service.py

```diff
@@ -114,6 +114,13 @@ def create_block(self, name, type, config=None):
             blk.save()
         return blk
 
+    def hasblock(self, block):
```

Comment:
I like the idea of a blocks property, I will implement it.

I would prefer a dict, that way you can do `name in service.blocks`. If you want to know if the object is in it: `block in service.blocks.values()`